components/EmotionList.js

```diff
@@ -0,0 +1,38 @@
+import { emotionData } from "@/lib/Data";
+import styled from "styled-components";
+
+const StyledUl = styled.ul`
```

Comment:
You could think about to rename the styled components to `StyledList` and `StyledListItem`, as we want to avoid shorthand notations for better readability